ddi/src/ddi_client_builder.cpp

```diff
@@ -99,6 +99,7 @@ namespace ddi {
         cli->defaultHeaders = defaultHeaders;
         cli->serverCertificateVerify = verifyServerCertificate;
         cli->authErrorHandler = authErrorHandler;
+        cli->isRunning = true;
```

Comment:
I would expect isRunning be true when the client is actually running. 
Wouldn't it be more reasonable to set this flag directly in HawkbitCommunicationClient::run() ?

Reply:
https://github.com/rtsoft-gmbh/up2date-cpp/pull/35/commits/c72f738b865e399773f7d7a6053d7435f033b126